app/database/queries/breeze_buddy/analytics.py

```
@@ -0,0 +1,282 @@
+            COUNT(*) as total_calls,
+            COUNT(*) FILTER (WHERE lct.status = 'FINISHED') as finished_calls,
+            COUNT(*) FILTER (WHERE lct.outcome IS NOT NULL) as calls_with_outcome,
+            jsonb_object_agg(
+                COALESCE(lct.outcome, 'N/A'),
+                COUNT(*)
+            ) as outcome_breakdown
+        FROM "{LEAD_CALL_TRACKER_TABLE}" lct
+        {where_clause}
+        GROUP BY lct.request_id
+        HAVING lct.request_id IS NOT NULL;
+    """
```

Ruff (0.14.8)

Possible SQL injection vector through string-based query construction

(S608)